js/onoffice-copycode.js

```diff
@@ -0,0 +1,3 @@
+jQuery(document).ready(function($){
+	new ClipboardJS('#button-copy');
```

Comment:
This library seems to use an [obsolete method](https://github.com/zenorocha/clipboard.js/issues/683). [`navigator.clipboard`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Interact_with_the_clipboard) has [good support](https://caniuse.com/mdn-api_navigator_clipboard), so could we use that instead?